src/components/toolbar/toolbar.js

```diff
@@ -0,0 +1,24 @@
+import template from "./toolbar.html";
+
+export class ToolbarComponent {
+  constructor(mountPoint, onMenuClickListener) {
+    this.mountPoint = mountPoint;
+    this.onMenuClickListener = onMenuClickListener;
```

Comment:
if `onMenuClickListener ` is a prop -- you should pass it via `props`

```js
constructor(mountPoint, props) {
 this.props = props
}
```

and `onMenuClickListener ` will be inside props